test/membership_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+class MembershipTest < Minitest::Test
+  def start_date
+    '2015-11-27'
+  end
+
+  def end_date
+    '2015-12-31'
+  end
```

Comment:
Each of these are only used in a single place, so it's a bit odd to create methods for them…